Industrial-IoT.sln

```diff
@@ -258,7 +234,15 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Serial
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Serializers.MessagePack.Tests", "common\src\Microsoft.Azure.IIoT.Serializers.MessagePack\tests\Microsoft.Azure.IIoT.Serializers.MessagePack.Tests.csproj", "{01A2DD25-32E4-424C-82E8-9C7111C64AAE}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Test.Scenarios", "api\src\Microsoft.Azure.IIoT.Api\tests\Microsoft.Azure.IIoT.Test.Scenarios.csproj", "{5B81D952-2E41-4704-8D22-2D817D5F94E6}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.AspNetCore.SignalR", "common\src\Microsoft.Azure.IIoT.AspNetCore.SignalR\src\Microsoft.Azure.IIoT.AspNetCore.SignalR.csproj", "{CF79D44A-E79C-42D6-BDCA-1DDFD7108AB5}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Services.OpcUa.Events.Tests", "services\src\Microsoft.Azure.IIoT.Services.OpcUa.Events\tests\Microsoft.Azure.IIoT.Services.OpcUa.Events.Tests.csproj", "{6CFFAE7C-5943-4954-B8BF-9C455AF7F071}"
+EndProject
```

Comment:
too many files for codeflow :-( I don't see the changed files in /services...

Reply:
Codeflow works just fine.

Reply:
which version do you have? I see only files under api/src

---
In reply to: [404276365](https://github.com/Azure/Industrial-IoT/pull/375#discussion_r404276365) [](ancestors = 404276365)